apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
+import PlansGridFSE from '../../../../editor-plans-grid/src/plans-grid-fse';
+import './styles.scss';
+
+const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep } ) => {
+	const handleBack = () => {
+		onPrevStep && onPrevStep();
+	};
+
+	return (
+		<LaunchStep className="domain-step">
+			<div className="launch-step__header">
+				<div className="launch-step__heading">
+					<h1 className="launch-step__title">{ __( 'Choose a plan', 'full-site-editing' ) }</h1>
+					<p className="launch-step__subtitle">
+						{ __(
+							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.' )` *ES Score: 11*